README.md

```diff
@@ -92,7 +92,21 @@ if (!global.Intl) {
 
 _note: the locale data is required for the polyfill to function when using it in a browser environment, in the example above, the english (`en`) locale is being required along with the polyfill itself._
 
+If you're using [webpack 2], instead of `require.ensure(...)` use `System.import`:
+
+```javascript
+// ...
+
+    Promise.all([
+        System.import('intl'),
```

Comment:
@silvenon I tried to use it, but simply cannot get loaders to work as expected, even if they provided by webpack team